docs/providers/okta.md

```diff
@@ -0,0 +1,174 @@
+
+## Usage
+
+### 1. Install dependencies
+`npm install @nuxtjs/auth @nuxtjs/axios @nuxtjs/dotenv`
+
+<details> 
+  <summary>
+    <strong><em>Note</em></strong>: Here are instructions for using the pull request (PR) version.
+  </summary>
+
+1. add the following to your `package.json` `dependencies`:
+   ```
+   "@nuxtjs/auth": "git+https://github.com/metasean/auth-module.git#okta",
+   "@nuxtjs/axios": "^5.5.4",
+   "@nuxtjs/dotenv": "^1.3.0",
+   ```
+2. run `npm install`
+</details>
+
+### 2. Create an `.env` file with the following information
+```bash
+# Base URL construction:
+# https://${OKTA_DOMAIN}.${OKTA_URL}.com/oauth2/${OKTA_SERVER_ID}/v1
+
+
+#######################################
+#
+# MANDATORY nuxt.config.js variables
+#
+
+# OKTA_DOMAIN
+# the unique portion of your okta url, e.g.
+# https://dev-012345-admin.okta.com
+# nuxt.config.js - apply to `auth.strategies.okta.domain`
+OKTA_DOMAIN='dev-012345'
+
+# OKTA_CLIENT_ID
+# the Client ID set by Okta
+# Can be located under:
+# Applications >> <app> >> General >> Client Credentials
+# nuxt.config.js - apply to `auth.strategies.okta.client_id`
+OKTA_CLIENT_ID='0abcde1fgHIjk2lMn345'
+
+
+#######################################
+#
+# OPTIONAL nuxt.config.js variables
+# values shown below are the default values
+#
+
+# OKTA_URL
+# the base okta url
+OKTA_URL='okta.com'
+
+# OKTA_REDIRECT_CALLBACK_URI
+# is the uri Okta will redirect the user to upon login
+# _MUST_ also be set in Okta's
+# Applications >> <app> >> General >> Login redirect URIs
+# default value is '/implicit/callback'
+# nuxt.config.js: apply to `auth.redirect.callback`
+REDIRECT_CALLBACK_URI='/implicit/callback'
+
+# OKTA_REDIRECT_LOGOUT_URI
+# is the uri Okta will redirect the user to upon logout
+# _MUST_ also be set in Okta's 
+# Applications >> <app> >> General >> Logout redirect URIs
+# default value is '/'
+# nuxt.config.js - apply to `auth.redirect.logout`
+REDIRECT_LOGOUT_URI='/'
+
+# OKTA_SERVER_ID
+# the generated portion of the url between `oauth2/` and `/v1`.
+# e.g. `abcdef0ghi1JklMNO234` is the server id for https://dev-012345-admin.okta.com/oauth/abcdef0ghi1JklMNO234/v1/authorize?...
+# Can be located under:
+# API >> Authorization Servers
+# default value is 'default'
+# nuxt.config.js - apply to `auth.strategies.okta.server_id`
+OKTA_SERVER_ID='default'
+
+# OKTA_SCOPE
+# Oauth2 scopes that are supported by the app
+# _MUST_ also be included in Okta's 
+# Applications >> <app> >> General >> Scopes
+# default value is 'openid profile email'
+# nuxt.config.js - apply to `auth.strategies.okta.scope`
+OKTA_SCOPE='openid profile email'
+```
+
+### 3. Add the following to the `nuxt.config.js` file
+```js
+
+require('dotenv').config();
+
+export default {
+  modules: [
+    '@nuxtjs/axios',
+    '@nuxtjs/auth',
+    ['@nuxtjs/dotenv', { /* https://github.com/nuxt-community/dotenv-module#options */ }]
+  ],
+
+  auth: {
+    redirect: {
+      callback: process.env.REDIRECT_CALLBACK_URI, // optional
```

Comment:
Maybe as a demo we can simply put defaults and comments here? + As of implicit flow, I don't see any point obscuring configuration with a dotenv file by default unless there is a deployment requirement to have different values for prod which is out of scope of this intro example or can be mentioned in footer to use either environment fallback like ` '/' || process.env.OKTA_REDIRECT_CALLBACK_API` or node config package or dotenv